tools/Common.props

```diff
@@ -10,7 +10,7 @@
     <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
     
     <!-- All MonoMod projects (by default) target these frameworks -->
-    <TargetFrameworks>net35;net452;netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0;net7.0;netstandard2.0;net35;net452</TargetFrameworks>
```

Comment:
There should probably be a comment here explaining this ordering. I'd personally also prefer if it were strictly newest -> oldest, though that's not especially important.

Reply:
I've put .NET 6.0 first considering it's an LTS version, and .NET 7.0 isn't fully supported yet from what I remember jade saying.

Reply:
.NET 7 should be fully supported, at least in RuntimeDetour. The only current limitation is around MacOS. Unless you're talking about support in, say, Rider, which I know is incomplete.